framework/src/modules/chain/index.js

```diff
@@ -66,32 +66,85 @@ module.exports = class ChainModule extends BaseModule {
 
 	get actions() {
```

Comment:
We should only make public few actions that are required in the current protocol imo. Everything else shouldn't be exposed.

Reply:
Made everything `false` except for `blocks` and `blocksCommon`.

Reply:
getTransaction and getSignarures also should be public